qa/L0_config_search/test_config_generator.py

```diff
@@ -70,7 +70,7 @@ def generate_search_disable(self):
         total_param_count = self._calculate_total_params(
             concurrency_count, instance_count
         )
-        self._write_file(total_param_count, 1, 2, 1, model_config)
+        self._write_file(total_param_count, total_param_count, 2, 2, model_config)
```

Comment:
Given that we no longer treat remote mode separately, do we need to pass a separate param counts for remote mode? They seem to be the same anyway.

Reply:
No, we don't need to. I've removed them.